nucliadb_models/src/nucliadb_models/internal/shards.py

```diff
@@ -63,6 +63,7 @@ class ShardReplica(BaseModel):
 
 class ShardObject(BaseModel):
     shard: str
+    nidx_shard_id: Optional[str]
     replicas: List[ShardReplica]
```

Comment:
can be removed?

Reply:
looks like it can be removed but it's annoying and breaks some tests. Maybe for the second pass?